syncer/syncer.go

```diff
@@ -219,6 +221,8 @@ func (s *Syncer) IsBeaconSynced(epoch types.EpochID) bool {
 func (s *Syncer) Start(ctx context.Context) {
```

Comment:
you don't have to pass in ctx either?

Reply:
Removed ctx from `Start`, since it is a relatively small change and related to fixing some tests for windows.

I don't want to do this in too many places though, since this PR should only as much as is needed to get the windows build / unit tests running